CONTRIBUTING.md

```diff
@@ -1,71 +1,73 @@
-# Contributing to OCamlFormat
+# OCamlFormat Contribution Guide
+
+Welcome and thank you for your interest in contributing to OCamlFormat. This contribution guide will help you through the contribution process.
 
 ## Reporting Issues
 
-If you encounter a problem when using OCamlFormat or if you have any questions, please open a [GitHub issue](https://github.com/ocaml-ppx/ocamlformat/issues/).
+If you encounter an issue with OCamlFormat, or if you have inquiries, we encourage you to open a [GitHub issue](https://github.com/ocaml-ppx/ocamlformat/issues/). Please follow the steps below:
 
-1. Check first if your issue has already been [reported](https://github.com/ocaml-ppx/ocamlformat/issues/).
-2. Include the version of OCamlFormat you are using (`ocamlformat --version`).
-3. Include a (preferably short) source file which demonstrates the issue.
-4. Describe the expected and actual behavior.
-5. Do not unsubscribe from the issue until it is closed, the maintainers may ask for your feedback.
+1. First, verify if your issue hasn't already been [reported](https://github.com/ocaml-ppx/ocamlformat/issues/).
+2. Specify the OCamlFormat version you're using (`ocamlformat --version`).
+3. Include a (preferably short) source file that replicates the issue.
+4. Clearly define the expected and actual behavior.
+5. Remain engaged with the issue until closure as your feedback might be needed.
 
-When acknowledged, the project maintainers will add [labels](#ocamlformat-labels) to your issue throughout its lifespan.
+Once your issue is acknowledged, our maintainers will apply relevant [labels](#ocamlformat-labels) to track its status.
```

Comment:
Talking about the labels seems unnecessary. Perhaps it can be removed to be straight to the point ? (same in pull requests)

Reply:
I've removed the mentions that maintainers use labels.

Reply:
Sounds good to me